templates/flavors/clusterclass-kubeadm/ReplacementTransformer.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Do we want to capitalize the first letter of filename? Not sure if we really intend to do that as other files have first letter as small case.

Reply:
good catch, just pushed up a change to fix that